File: mambo-protocol/src/main/scala/org/mambo/protocol/messages/game/context/mount/MountSterilizeRequestMessage.scala





















// Generated on 05/03/2013 10:34:41
// Scala version by Blackrush
package org.mambo.protocol.messages

import org.mambo.core.io._
import org.mambo.protocol.{Message => BaseMessage, _}
import org.mambo.protocol.types._
import org.mambo.protocol.enums._

trait MountSterilizeRequestMessage extends MessageDeserializer {
  val messageId = 5962

  protected trait Message extends BaseMessage with Serializable {
    val messageId = MountSterilizeRequestMessage.messageId


    def serialize(buf: Buffer) {
    }

  }

  protected def create(): Target = throw new IllegalAccessException

  def deserialize(buf: Buffer) = {

    create()
  }

}

object MountSterilizeRequestMessage extends MountSterilizeRequestMessage {
  case class MessageImpl() extends Message
  type Target = MessageImpl

  override def create() = MessageImpl()

  def apply() = create()
  def unapply(msg: MessageImpl) = MessageImpl.unapply(msg)
}

File: mambo-shared/mambo-protocol/mambo-protocol-client/src/main/java/org/mambo/protocol/client/messages/MountSterilizeRequestMessage.java


// Generated on 12/14/2012 18:44:07
package org.mambo.protocol.client.messages;

import java.util.*;
import org.mambo.protocol.client.types.*;
import org.mambo.protocol.client.enums.*;
import org.mambo.protocol.client.*;
import org.mambo.core.io.*;

public class MountSterilizeRequestMessage extends NetworkMessage {
    public static final int MESSAGE_ID = 5962;
    
    @Override
    public int getNetworkMessageId() {
        return MESSAGE_ID;
    }
    
    
    public MountSterilizeRequestMessage() { }
    
    
    @Override
    public void serialize(DataWriterInterface writer) {
    }
    
    @Override
    public void deserialize(DataReaderInterface reader) {
    }
    
}
